custom/icds/templates/icds/messaging/indicators/mar/aww_vhnd_survey.txt

```diff
@@ -0,0 +1 @@
+As per the record, VHSND could not happen at your centre in the previous month. Please ensure this is conducted on regular basis every month.
```

Comment:
Is this supposed to be in Marathi?

Reply:
@esoergel I believe we're still waiting on a translation for this one yet.